state-chain/runtime-utilities/macros/Cargo.toml

```diff
@@ -6,8 +6,11 @@ edition = '2021'
 [lib]
 proc-macro = true
 
+[lints]
+workspace = true
+
 [dependencies]
 # proc-macro-crate = '0.1.5'
 # proc-macro-error = '1.0.4'
 quote = '1.0'
-syn = '1.0'
\ No newline at end of file
+syn = '1.0'
```

Comment:
Having an extra empty line at the end of files in nice, because it makes diffs nicer, when you add new lines.

Reply:
Not actually sure how this was added 😅 
Presumably some new cargo fmt feature?